components/clarinet-deployments/Cargo.toml

```diff
@@ -12,26 +12,36 @@ serde_yaml = "0.8.23"
 clarity_repl = { package = "clarity-repl", path = "../clarity-repl", default-features = false, optional = true }
 clarinet_files = { package = "clarinet-files", path = "../clarinet-files", default-features = false }
 chainhook_types = { package = "chainhook-types", path = "../chainhook-types-rs" }
-stacks_rpc_client = { package = "stacks-rpc-client", path = "../stacks-rpc-client", optional = true  }
+stacks_rpc_client = { package = "stacks-rpc-client", path = "../stacks-rpc-client", optional = true }
 
 # CLI
 reqwest = { version = "0.11", default-features = false, optional = true, features = [
     "json",
     "rustls-tls",
 ] }
```

Comment:
if `reqwest` is always used, then we can remove `optional = true` here, and remove `reqwest` from the `wasm` and `onchain` feature gates.

Reply:
Alright sounds good I'll do that. I wasn't sure why it was optional in the first place